common/src/main/java/com/wynntils/features/ItemStatInfoFeature.java

```diff
@@ -28,11 +29,28 @@
 import net.minecraft.network.chat.TextComponent;
 import net.minecraft.world.item.ItemStack;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
 
-public class ItemStatPercentageInfoFeature extends Feature {
+public class ItemStatInfoFeature extends Feature {
     private static final Pattern ITEM_STATUS_PATTERN =
             Pattern.compile("([+\\-])(\\d+)(%|\\stier|/3s|/4s|/5s|)\\*{0,3}\\s(.+)");
 
+    // TODO: Replace these with configs
+    // Possible variables: %percentage% %minmax% %reidchance%
+    private static final String MAIN_FORMAT_STRING = "%percentage%";
```

Comment:
If we're re-inventing the info strings, maybe chose a wiser format this time? Using asymmetric delimiters help parsing, both for human readers and for the code. In other words, `{percentage}` is a better format string than `%percentage%`.

Reply:
I agree, let's refactor this.

Reply:
I've also been thinking about not really having "info" variables, as much as having "variables". To expand a bit, we should have some kind of standardized registry mapping String names to lambdas returning a value. We could then use this "source of truth" in our code, but we could also allow users to access it through the "info variable" syntax. It's not a big change in the code itself, it's more of a conceptual change. I'm not sure if such a thing should be part of this PR (probably not, better to have small, dedicated PRs) but I'd like to see something like that at some point in time for Artemis.

Reply:
Sounds good, however note that these are "item based" info variables so it wouldn't really be useful here. However cool idea for  normal info variables.